packages/edit-post/src/components/block-settings-menu/plugin-block-settings-menu-item.js

```diff
@@ -40,7 +40,6 @@ const shouldRenderItem = ( selectedBlocks, allowedBlocks ) =>
  * @param {string}                [props.role]          The ARIA role for the menu item.
  *
  * @example
- * <caption>ES5</caption>
```

Comment:
@nosolosw, wasn't it necessary for the Block Editor Handbook to add tabs to switch between two versions of the example?

Reply:
Hmm, I don't see it having any impact on the generated output. So it looks like it was obsolete 😄

Reply:
I commented in another thread above I can't find now :sweat_smile: That was the original goal but don't remember we ever implemented it.